bindings/libdnf5/advisory.i

```diff
@@ -38,6 +38,9 @@
 
 #define CV __perl_CV
 
+#define LIBDNF_API
+#define LIBDNF_LOCAL
```

Comment:
Can we define it once, e.g. in `shared.i`?

Reply:
OK. I moved the macro definition to `shared.i`.